### ISSUE TITLE
No available formula with the name "pack"

### ISSUE THREAD

**Issue by dentarg** [heroku/buildpacks-ruby#249]
The README says:

>To use the currently deployed `heroku/ruby` buildpack locally, you'll need a copy of `pack` installed. On a mac you can run:
>
>     $ brew install pack

but when I try this it fails:

```
$ brew info pack
Error: No available formula with the name "pack". Did you mean packr, spack, hpack, mpack, ack or pick?
```

```
$ brew -v
Homebrew 4.2.4
$ brew update
Already up-to-date.
```

**Comment by dentarg** [comment]
@edmorley @schneems can you help out? how do I get `pack`?

**Comment by edmorley** [comment]
See:
https://buildpacks.io/docs/tools/pack/

@schneems Please can you fix the README (suggest linking to https://buildpacks.io/docs/tools/pack/ instead of inline the install steps, given there are other OSes, and also the current steps are wrong).

**Comment by edmorley** [comment]
The docs were fixed in #263.
